tests/integration/setup/helpers.js

```diff
@@ -187,27 +187,22 @@ const describeJsHandle = (jsHandle) => {
   }, jsHandle)
 }
 
-export const BOBBO_INSTALLED_APP_ID = 'second_agent'
-
 export const setupTwoChatters = async (scenario, createPage, callRegistry) => {
   // Tryorama: instantiate player conductor
   console.log('Setting up players on elemental chat...')
-  const [aliceConductor, bobboConductor] = await scenario.players([aliceConductorConfig, otherConductorConfig])
+  const [conductor] = await scenario.players([conductorConfig], false)
 
-  await scenario.shareAllNodes([aliceConductor, bobboConductor])
+  await conductor.startup()
 
-  aliceConductor.setSignalHandler(s => {
-    console.log('Alice Conductor Received Signal:', s)
-  })
-  bobboConductor.setSignalHandler(s => {
-    console.log('Bobbo Conductor Received Signal:', s)
+  conductor.setSignalHandler((_) => {
+    console.log('Conductor Received Signal:', _)
   })
 
   // Tryorama: install elemental chat on both player conductors
   const bundlePath = path.join(__dirname, 'bundle', 'elemental-chat.happ')
 
-  const aliceChatHapp = await aliceConductor.installBundledHapp({ path: bundlePath }, null, INSTALLED_APP_ID)
-  const bobboChatHapp = await bobboConductor.installBundledHapp({ path: bundlePath }, null, BOBBO_INSTALLED_APP_ID)
+  const aliceChatHapp = await conductor.installBundledHapp({ path: bundlePath }, null, INSTALLED_APP_ID)
+  const bobboChatHapp = await conductor.installBundledHapp({ path: bundlePath }, null, 'second_agent')
```

Comment:
I intentionally made this two conductors because I heard from holochain team that gossip within a conductor just doesn't work. You can work around that by setting the dht_single_arc tuning param, but that feels bad to me because then you're not actually testing gossip. In other words, at that point you could just have one agent and check that they see their own messages and it feels equivalent